_i18n/nb-no/resources/user-guides/cli_wallet_daemon_isolation_qubes_whonix.md

```diff
@@ -0,0 +1,149 @@
+{% include disclaimer.html translated="yes" translationOutdated="no" %}
```

Comment:
You translated an obsolete version of this guide, which was updated in November: https://github.com/monero-project/monero-site/commit/5c16cd56d06697159587141a6ce373fc95e71966#diff-cff12cc7262fa2e7095e79aa941a500914e6f56085cc479c944dd8978889f54a

Reply:
Fixed